atlasdb-cassandra/build.gradle

```diff
@@ -41,6 +41,9 @@ dependencies {
 
   explicitShadow group: 'org.jboss.marshalling', name: 'jboss-marshalling', version: '1.4.11.Final'
 
+  api 'com.google.protobuf:protobuf-java'
+  api 'com.palantir.common:streams'
+
   testImplementation project(path: ":atlasdb-client", configuration: "testArtifacts")
```

Comment:
discussed offline: we should be able to remove this.

Reply:
yep, removed it. also added api deps for all classes used in TableRenderer and related classes.